internal/configs/executionspace/config.go

```diff
@@ -0,0 +1,172 @@
+// Copyright Axis Communications AB.
+//
+// For a full list of individual contributors, please see the commit history.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Config interface for retreiving configuration options.
+type Config interface {
+	ServiceHost() string
+	ServicePort() string
+	StripPrefix() string
+	Hostname() string
+	LogLevel() string
+	LogFilePath() string
+	Timeout() time.Duration
+	KubernetesNamespace() string
```

Comment:
Why is there both a KubernetesNamespace and an ETOSNamespace? I assume they are equal

Reply:
They are now merged into one.